geode-gfsh/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/StatusRedundancyCommandDUnitTest.java

```diff
@@ -197,14 +197,24 @@ public void statusRedundancyWithIncludeRegionReturnsErrorWhenAtLeastOneIncludedR
   }
 
   @Test
-  public void statusRedundancyReturnsEmptyRegionStatusAsNonRedundantCopies() {
+  public void statusRedundancyReturnsRegionWithNoBucketsCreatedStatusAsNoRedundantCopies() {
     createRegion();
     String command = new CommandStringBuilder(STATUS_REDUNDANCY).getCommandString();
     CommandResultAssert commandResult = gfsh.executeAndAssertThat(command).statusIsSuccess();
     verifyGfshOutput(commandResult, Collections.singletonList(EMPTY_REGION), new ArrayList<>(),
         new ArrayList<>());
   }
 
+  @Test
+  public void statusRedundancyReturnsEmptyRegionStatusAsSatisfied() {
```

Comment:
To be more accurate, this test name should just say that an empty region reports redundancy status correctly, since an empty region will not report as "satisfied" if there aren't enough servers to host that many copies, or if redundancy isn't actually satisfied due to restarted servers etc. Really, what the test is showing is that an empty region behaves no different from a region with entries in it.

Reply:
I agree, good catch